src/lib/utils/FileFactory.ts

```diff
@@ -20,6 +20,7 @@ export class FileFactory {
             let sourcePath = path.resolve(path.join(this.sourcePath, fileName));
             let fileContents = fs.readFileSync(sourcePath, 'utf8');
             let destPath = this.frameworkFiles[fileName];
+            //BRON_AST_EDIT_HERE
```

Comment:
@georgejecook, similar to my question from rooibos. 

These framework files are static, right? As in, they don't change as part of every watch cycle? Rather than adding (and re-parsing) them every time in beforeProgramTranspile, and then removing them every afterProgramTranspile, I'm thinking it's ok to leave this code as-is. That'll save time not re-parsing them every time.

Reply:
No.. this is a little different to rooibos here. In this case the reflection file is generated allowing me to call constructors from strings. So this file is required to be in the code; but only needs to be update once, before going to the device. 

It's a huge if statement of class names to constructor functions.